### PR TITLE
153 bug graph sdk adding redundant connection throws an error

### LINKED ISSUE #153: [bug] [graph-sdk] Adding redundant connection throws an error
Using the npm bridge, when trying to add a connection that already exists to a graph using `applyActions`, an error is thrown.

Actual behavior: an error is thrown (npm bridge)

Expected behavior: operation is a no-op/silently fails

### PR DESCRIPTION
**Pull request by JoeCap08055**
# Goal
The goal of this PR is add an option to ignore errors on adding existing connections or removing non-existent connections from a user graph.

Closes #153 

# Discussion

# Checklist
- [x] Added option to export a graph for a single user
- [x] Added option to ignore redundant/missing connection errors on add/remove connection
- [x] Rust tests updated
- [x] FFI tests updated
- [x] Java tests updated
- [x] Node tests updated

### CODE REVIEW

**Comment by JoeCap08055** [comment]
Starting this draft PR so review can start. Items TBD:
* If we're not checking existing/missing connections in `apply_actions`, we need to add logic to `calculate_updates` to make sure we don't add redundant connections
* Update/add tests

File: java/lib/src/main/java/io/amplica/graphsdk/Graph.java [hunk 1]
```diff
@@ -58,16 +57,19 @@ public void applyActions(Actions actions) throws BaseGraphSdkException {
...
+    public List<Updates.Update> forceRecalculateGraph(long dsnpUserId)
+            throws BaseGraphSdkException, InvalidProtocolBufferException {
         try (NativeHandleGuard guard = new NativeHandleGuard(this)) {
             var raw = Native.forceCalculateGraphs(guard.nativeHandle(), dsnpUserId);
             return Updates.parseFrom(raw).getUpdateList();
         }
     }
 
-    public List<DsnpGraphEdges.DsnpGraphEdge> getConnections(long dsnpUserId, ConnectionType connectionType, boolean includePending) throws BaseGraphSdkException, InvalidProtocolBufferException {
+    public List<DsnpGraphEdges.DsnpGraphEdge> getConnections(long dsnpUserId, ConnectionType connectionType,
```
Comment (aramikm) [inline java/lib/src/main/java/io/amplica/graphsdk/Graph.java:68]: Please don't forget to add that new api that exports updates only for a certain user to JVM

**Review by aramikm — APPROVED** [review]
Looks good, Added one comment!

**Review by saraswatpuneet — APPROVED** [review]

**Review by aramikm — APPROVED** [review]
Looks great!

**Pull request merged**